benchmarks/commonsense_qa/commonsense_qa.py

```diff
@@ -0,0 +1,48 @@
+"""
+CommonsenseQA (CSQA): A Question Answering Challenge Targeting Commonsense Knowledge
+
+Alon Talmor, Jonathan Herzig, Nicholas Lourie, Jonathan Berant
+https://arxiv.org/pdf/1811.00937v2
+
+# eval w/ 500 randomly selected samples
+inspect eval csqa.py --limit 500
+"""
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.dataset._sources.hf import hf_dataset
+from inspect_ai.model._generate_config import GenerateConfig
+from inspect_ai.scorer import choice
+from inspect_ai.solver import multiple_choice
+
+
+def record_to_sample(record):
```

Comment:
We typically put the `record_to_sample` function below the task function.

Reply:
I can move it, but I just checked some other benchmarks again like [mmli](https://github.com/UKGovernmentBEIS/inspect_ai/blob/main/benchmarks/mmlu.py), [gsm8k](https://github.com/UKGovernmentBEIS/inspect_ai/blob/main/benchmarks/gsm8k.py) or [hellaswag](https://github.com/UKGovernmentBEIS/inspect_ai/blob/main/benchmarks/hellaswag.py). For all of those the `record_to_sample` function seems to be above the task function.

Reply:
Okay, nevermind, just saw the _newer_ benchmark implementations under the subdirectories have the function under the task function. I'll move it.